src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -356,15 +356,25 @@ short Terminal::GetBufferHeight() const noexcept
 }
 
 // _ViewStartIndex is also the length of the scrollback

```

Comment:
comment of function name no longer matches function's name after rename

Reply:
changed